microsoft-edge/devtools-guide-chromium/whats-new/2024/02/devtools-122.md

```diff
@@ -0,0 +1,90 @@
+---
+title: What's New in DevTools (Microsoft Edge 122)
+description: "Z-index and DOM tabs in 3D View tool working correctly. Correct Hide debugger or Show debugger icon in Sources tool. Source map support in Coverage tool. Advanced filtering in Bottom-Up, Call Tree, and Event Log tabs in Performance tool. Window Controls Overlay section for PWAs, in Application tool's Manifest section. And more."
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: devtools
+ms.date: 02/22/2024
+---
+# What's New in DevTools (Microsoft Edge 122)
+
+[!INCLUDE [Microsoft Edge team note for top of What's New](../../includes/edge-whats-new-note.md)]
+
+
+<!-- ====================================================================== -->
+## Icons are fixed in DevTools UI
+
+<!-- Subtitle: In recent versions of Microsoft Edge, icons were missing or were colored incorrectly. In Microsoft Edge 122, this issue has been resolved. -->
+
+There was a bug where numerous any icons were missing, had the wrong background color, or didn't work with high contrast mode throughout DevTools. In Microsoft Edge 122, all these icons have been fixed and are compatible when high contrast mode is enabled.
+
+<!-- ====================================================================== -->
+## Keyboard shortcut to change panels is now working correctly
+
+<!-- Subtitle: In previous versions of Microsoft Edge, the Ctrl + ]/[ keyboard shortcut wasn't working correctly. In Microsoft Edge 122, this issue has been resolved. -->
+
+There was a bug with the Ctrl + ]/Ctrl + [ keyboard shortcut where the next tool in the Activity Bar wasn't selected and the Activity Bar wouldn't update so it looked like the previous tool was selected. For example, starting on the Elements panel:
+![Initial state, in Elements tool](./devtools-122-images/keyboard-shortcut-initial-state.png)
+
+Using the shortcut to move across tools, the Elements tab is still highlighted, but the Network tool is displayed:
+![Incorrect tool and tab name displayed](./devtools-122-images/keyboard-shortcut-error.png)
+
+This issue has been resolved with Microsoft Edge 122 and works as expected.
+
+<!-- ====================================================================== -->
+## Context menu is fixed
```

Comment:
there's no such thing as "the DevTools context menu".  Do you mean that the **Customize and control DevTools** (...) button doesn't work?  That button seems to work; can't repro in Stable 121.